packages/expo-ui/src/swift-ui/ContextMenu/index.tsx

```diff
@@ -84,36 +67,24 @@ export function Preview(props: { children: React.ReactNode }) {
  * - Android does not support showing a `Picker` element in the context menu.
  */
 function ContextMenu(props: ContextMenuProps) {
-  const eventHandlersMap: EventHandlers = {};
-  const initialChildren = Children.map(
-    props.children as any,
-    (c: { type: { tag: string }; props: { children: React.ReactNode } }) =>
-      c.type.tag === Items.tag ? c.props.children : null
-  );
-  const processedElements = useMemo(
-    () => transformChildrenToElementArray(initialChildren, eventHandlersMap),
-    [initialChildren]
-  );
-
-  const createEventHandler =
-    (handlerType: string) => (event: NativeSyntheticEvent<{ contextMenuElementID: string }>) => {
-      const handler = eventHandlersMap[event.nativeEvent.contextMenuElementID]?.[handlerType];
-      handler?.(event);
-    };
-
-  return (
-    <MenuNativeView
-      elements={processedElements}
-      onContextMenuButtonPressed={createEventHandler('onPress')}
-      onContextMenuSwitchValueChanged={createEventHandler('onValueChange')}
-      onContextMenuPickerOptionSelected={createEventHandler('onOptionSelected')}
-      {...props}
-    />
-  );
+  return <MenuNativeView {...props} />;
 }
 
 ContextMenu.Trigger = Trigger;
 ContextMenu.Preview = Preview;
 ContextMenu.Items = Items;
 
-export { ContextMenu };
+/**
+ * @deprecated Use `ContextMenu` component as Submenu instead.
+ */
+const Submenu = (props: SubmenuProps) => {
```

Comment:
Marked this deprecated. One can use `<ContextMenu />` as child, and it works. Better to reuse the API (nesting/dynamic is much easier/cleaner)